### DISCUSSION TITLE
Put tooltip on top of all other elements.

### DISCUSSION THREAD

**Discussion by Uncasted** [saadeghi/daisyui#1047] [Q&A]
Hello.

I'm using daisyUI for an application that I'm currently developing (<a href="https://github.com/uncasted/sencrypt">Sencrypt<a/>). And there's one problem.

![image](https://user-images.githubusercontent.com/100163644/186037998-afe1c52f-1f0e-48f4-8299-2d8ae160860c.png)

As you can see, I'm using a tooltip for the clipboard, but unfortunately, it's hidden behind the container, Is there a way I can change the positioning of the tooltip so that it's on top of it?

**Comment by saadeghi** [comment]
Please provide a code example. I think your container has some overflow class that hides overflowing elements but I'm not sure.

**Comment by Fish1** [comment]
@Uncasted Hey, sorry to reply after a year... but have you found any solution for this?

**Comment by saadeghi** [comment]
You can use a JS library like **Floating UI** to automatically handle positions because that's something CSS can not do (yet)  
https://github.com/saadeghi/daisyui/discussions/1695#discussioncomment-6006234
